UnityProject/Assets/Scripts/Player/PlayerScript.cs

```diff
@@ -176,6 +175,15 @@ public ChangelingMain Changeling
 		}
 	}
 
+	private Faith currentFaith = null;
+	public Faith CurrentFaith
+	{
+		get => currentFaith;
+		private set => currentFaith = value;
```

Comment:
what's the need of the field if you have the private set here?

Reply:
idk, rider automatically did this

Reply:
You can merge them